src/index.js

```diff
@@ -7,9 +7,23 @@ window.onload = () => {
   const parser = new DOMParser();
   const wrapper = document.querySelector('.wrapper');
   const listsContainer = document.querySelector('.lists-container');
+  const remCompTaskBtn = document.querySelector('.clear-btn');
   const addBtn = document.querySelector('.add-btn');
   let listArray = [];
 
+  // localStorage.removeItem('listsKey');
+  const taskCompleteUpdate = (e, index, t) => {
+    if (e.target.checked) {
+      listArray[index].completed = true;
+      t.style.textDecoration = 'line-through';
+    } else {
+      listArray[index].completed = false;
+      t.style.textDecoration = 'none';
+    }
+
+    localStorage.setItem('listsKey', JSON.stringify(listArray));
+  };
```

Comment:
[REQUIRED] Your `clear all completed` button removes both checked and unchecked elements It only returns  the unchecked element to the screen on page refresh 

![todo1](https://user-images.githubusercontent.com/62938735/194717084-ea52d13b-35f7-4f04-b14a-a55168b53d3c.gif)